src/server/helpers/diffFormatters/base.js

```diff
@@ -76,7 +76,7 @@ export function formatEndedChange(change) {
 }
 
 export function formatTypeChange(change, label) {
-	return formatChange(change, label, (side) => side && [side.label]);
+	return formatChange(change, label, (side) => (typeof side === 'string' ? [side] : side && [side.label]));
```

Comment:
This is getting a bit hard to read, how about breaking it down again like this?
```suggestion
	return formatChange(change, label, (side) => {
	if(typeof side === 'string'){
	    return [side];
	}
	else if(side) {
	    return [side.label]
	}
	});
```